### ISSUE TITLE
PDF Input not working

### ISSUE THREAD

**Issue by ahazatif** [ssraza21/textifyi-issues#4]
I bought a premium subscription on your textifyi website to upload a larger file for digitization but even after buying the subscription, I'm still getting a message saying that I need a subscription to upload a larger file.

**Comment by ssraza21** [comment]
Salaam! 

I just made an adjustment - do you mind checking if it works? Also, I've set up a 300 page limit since some users were having issues for more pages. I'm working on a fix but if you do have an issue then please split the PDF

And if you have any issues, feel free to message me or send me the file and I can give you the output inshaAllah.

**Comment by ahazatif** [comment]
Wassalam,

Your fix worked! However, now when I try to create a searchable pdf, after having adjusted and tested it, I click the create pdf button and it loads but then it does not give me a final pdf to download.

JazakaAllah Khayr

**Comment by ahazatif** [comment]
Also I received this error when trying to translate my pdf: Translation failed. Error: AxiosError: Request failed with status code 500

**Comment by ssraza21** [comment]
I know its not obvious but the translate _tab_ is for images.
Use this to translate: https://www.texti.fyi/en/translate . note that itll take a bit (around an hour).

And how many pages was your PDF? and did you turn test mode off? Feel free to send it to me and I'll pass back the searchable PDF

Apologies for all the issues, I've upped your credits back to 300. Please send me the searchable PDF file and let me know if the translation works!

**Comment by ahazatif** [comment]
Yeah I don't really need the translation. I was just trying that feature out.

My pdf was 266 pages. Yes, I did turn off test mode.

**Comment by ahazatif** [comment]
I tried attaching the pdf here but it's not going through for some reason.

**Comment by ssraza21** [comment]
You can send it to me at contact@texti.fyi 

If you send it right now, I can send it back iA before I go to work

**Comment by ahazatif** [comment]
Sent.
